lang/en.json

```diff
@@ -84,6 +84,21 @@
       "title": "Multiple Choice Question",
       "description": "A Multiple Choice Question must be answered here. You can define one correct Answer and an arbitrary amount of wrong answers."
     },
+    "gaptext": {
```

Comment:
make this `gapText` to be consistent with the rest

Reply:
Fixed in 1690f68